dwave/gate/qir/__init__.py

```diff
@@ -24,14 +24,14 @@
 except ImportError:  # pragma: no cover
     pyqir_installed = False
 else:
-    # pyqir>=0.8 required
+    # pyqir==0.9.0 required
     from importlib.metadata import version
+    from packaging.version import parse
 
-    def parse(version: str) -> Tuple[str, str, str]:
-        "Parse a MAJOR.MINOR.PATCH version string into a tuple."
-        return tuple(version.split("."))[:3]
+    pyqir_installed = True
 
-    pyqir_installed = parse(version("pyqir")) >= parse("0.8.0")
+    if parse(version("pyqir")) != parse("0.9.0"):  # pragma: no cover
+        raise ImportError(f"PyQIR v0.9.0 required, found {version('pyqir')}")
```

Comment:
Maybe I'm missing something, but why isn't `pyqir` included as a dependency anywhere, not even in `extras`?

Reply:
Not really, it's set as an optional dependency and, while it could (and maybe should) be part of extras, I wanted to check its compatibility before running any QIR compilation, e.g., in case `pyqir` is installed separately.

Reply:
No, the check is fine, I'm just wondering how are users supposed to know they need to install `pyqir` manually? It's not in package deps, and it's not in readme (docs?).

Reply:
Ah, I see. That's a good point. 🤔

Reply:
@randomir Added PyQIR as an extra and also added a note in the installation instructions.